### ISSUE TITLE
Display support email more prominently

### ISSUE THREAD

**Issue by JLuse** [getsentry/sentry#60658]
### Problem Statement

A customer reported that it's difficult to find the support@sentry.io contact email in the platform. In cases where the support widget is not working, customers do not know how to reach out.

### Solution Brainstorm

Display the support email more prominently in platform.

### Product Area

Help

**Comment by brianthi** [comment]
We have a help flow on the left nav, which looks like it is able to directly create zendesk tickets in the app and add attachments as well. I feel like this is actually a better experience than having the email on the page somewhere. I do see it's slightly lengthy as to get to the flow you need to click the help button, type in something that's wrong, then see that there's a prompt to contact support at the bottom of the popup. Or is this the "Support widget" you're talking about?

**Comment by JLuse** [comment]
Yes, left-hand side, that flow is preferred. I think the customer was hoping for a way to have the email more exposed in case the there are any problems with that method, or just an alternative to get in touch.

**Comment by brianthi** [comment]
Got it, let me float this by design as it can be a quick solve. In the mean time i'll put it into our backlog
